src/cljs/harja/views/urakka/laadunseuranta/talvihoitoreitit_nakyma.cljs

```diff
@@ -0,0 +1,129 @@
+        "Lataa Excel-pohja"
+        "/excel/harja_talvihoitoreitit_pohja.xlsx"]]]
+     (if (empty? talvihoitoreitit)
+       [:div "Ei talvihoitoreittejä. Aloita tuomalla reitit käyttäen excel-tiedostoa."]
+       (doall
+         (for [rivi talvihoitoreitit
+               :let [reittien-maara (count (:reitit rivi))
+                     auki? (contains? (:talvihoitoreittien-tilat app) (:id rivi))]]
+           ^{:key (:id rivi)}
+           [:<>
+            [:div.flex-row.venyta.otsikkokomponentti {:class (str "" (when (> reittien-maara 0) " klikattava"))
+                                                      :on-click #(when (> reittien-maara 0) (e! (tiedot/->AvaaTalvihoitoreitti (:id rivi))))
+                                                      :data-cy (str "avaa-reitti-" (:nimi rivi))}
+             ;; Nuoli
+             [:div.basis48.nogrow
+              (when (> reittien-maara 0)
+                (if auki?
+                  [ikonit/navigation-ympyrassa :down]
+                  [ikonit/navigation-ympyrassa :right]))]
+
+             ;; Nimi
+             [:div.basis256.nogrow.shrink3.rajaus
+              [:div {:style {:display "flex"}}
+               [:span {:style {:margin-top "auto" :margin-bottom "auto" :margin-right "5px" :width "12px" :height "12px" :background-color (:vari rivi)}}]
+               [:div.semibold.musta.body-text (str (:nimi rivi))]]
+              [:div.body-text.harmaa (:pituus rivi)]]
+
+             [:div.basis384.grow2.shrink3.rajaus
+              [:div.body-text.semibold.musta "Hoitoluokkien osuudet reitillä (km)"]
+              [:div {:style {:display "flex"}}
+               (doall (for [h (:hoitoluokat rivi)]
+                        ^{:key (hash (str "hoitoluokka-" h))}
+                        [:div
+                         [:div.body-text.musta {:style {:padding-right "10px"}} (:hoitoluokka h)]
+                         [:div.small-text.musta {:style {:padding-right "10px"}} (fmt/desimaaliluku-opt (:pituus h) 2) ]]))]]
+
+             [:div.basis384.grow2.shrink3.rajaus
+              [:div.body-text.semibold.musta "Kalusto (kpl)"]
+              [:div {:style {:display "flex"}}
+               (doall (for [kalusto (:kalustot rivi)]
+                        ^{:key (hash kalusto)}
+                        [:div
+                         [:div.body-text.musta {:style {:padding-right "10px"}} (:kalustotyyppi kalusto)]
+                         [:div.small-text.musta {:style {:padding-right "10px"}} (:kalustomaara kalusto) ]]))]]
+
+             [:div.basis256.grow2.shrink2
+              [:div.body-text.strong.musta ""]
+              ;; Näytä valittu rivi kartalla tai piilota se
+              [:<>
+               (if (contains? valitut-kohteet (:id rivi))
+                 (harja.ui.napit/avaa "Piilota kartalta" #(e! (tiedot/->PoistaValittuKohdeKartalta (:id rivi))))
+                 (harja.ui.napit/avaa "Näytä kartalla" #(e! (tiedot/->LisaaValittuKohdeKartalle (:id rivi)))))]]]
```

Comment:
Ikonit näissä napeissa on erilaisia designissa, sekä hieman eri keskitys, mutta minun  puolesta kyllä aivan hyvä näin.